src/DependencyInjection/Configuration.php

```diff
@@ -16,10 +16,10 @@ class Configuration implements ConfigurationInterface
     /**
      * {@inheritDoc}
      */
-    public function getConfigTreeBuilder()
+    public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('zicht_url');
+        $treeBuilder = new TreeBuilder('zicht_url');
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
Nice. Dit zit in nog veel meer bundles, je kunt deze tegenwoordig vrij makkelijk terugvinden in de deprecations log van je profiler en dan zoeken naar "zicht". 

Dit is wel een fix die noodzakelijk is voor SF 5. Ben je van plan meer bundles langs te lopen?

Reply:
Tot nu toe is het redelijk random allemaal wat ik doe 😄 Er is op dit moment nog geen concreet plan zoals bij de upgrades van de bundles naar Sf 4.

> je kunt deze tegenwoordig vrij makkelijk terugvinden in de deprecations log van je profiler en dan zoeken naar "zicht".

Goeie tip. Ik zit op dit moment veel in de logs om dingen te checken, dus ik zie ze gewoon voorbij komen en pak het dan mee.